docs/checks.md

```diff
@@ -303,5 +303,57 @@ In line 3, column 14: PROTO and MODULE inputs should be in seperate groups. [imp
 In line 3, column 21: "random" is already defined. [imports]
 ```
 
+## expression-parantheses
+Use parathenthesis to highlight the order when there are multiple Boolean operators of different types.
 
+**Example**
+```sql
+-- Here is ok because they are the same type.
+SELECT D WHERE a AND b AND c;
+
+-- Here is an invalid usage of operators.
+SELECT D WHERE a AND b OR c; -- line 5
+
+-- It can be fixed by adding paratheses.
+SELECT D WHERE a AND (b OR c);
+```
+
+**Linter Output**
+```
+In line 5, column 16: Use parantheses between consequtive AND and OR statements. [expression-parantheses]
+```
+
+## count-star
+Prefer COUNT(*) over COUNT(1). They have the same functionality.
+
+**Example**
+```sql
+-- Simple Example
+SELECT COUNT(1); -- line 2
+SELECT COUNT(*); -- line 3
+```
 
+**Linter Output**
+```
+In line 2, column 15: Use COUNT(*) instead of COUNT(1) [count-star]
+```
+
+## keyword-identifier
+If the name of an identifier(column name, parameter name etc.) is also a SQL keyword. Change the name or use escape characters(`).
```

Comment:
Do not use ZetaSQL keywords as identifiers (aliases, function arguments, etc.). Change the name or escape with backticks (`).

Btw, I don't remember what exactly you're checking there. I thought you've limited this check to aliases only.

Reply:
It was another update that makes the check "column-name" aliases only. This is a seperate check that warns keyword-named identifiers.